app/components/Search/index.tsx

```diff
@@ -6,6 +6,7 @@ import { useIsLoggedIn } from 'app/reducers/auth';
 import { selectAutocompleteRedux } from 'app/reducers/search';
 import { useAppDispatch, useAppSelector } from 'app/store/hooks';
 import { Keyboard } from 'app/utils/constants';
+import FancyNodesCanvas from '../Header/FancyNodesCanvas';
```

Comment:
Not a fan of this, feels weird when it's not there before and then it's there when you open the search bar.

Not as noticeable on dark mode, but on light mode it feels weird weird

Thinking about it it might also be because the gray color of the FancyNodes is slightly different from the gray color that is now set as the background - so the entire background changes when the search bar is opened

Reply:
Anyways the fancy nodes feel even more noticeably old when I'm getting used to seeing the rest of the website without them - so opening the menu feels like an ugly blast from the past

Reply:
I agree - I vote to remove entirely